portfolio/src/main/webapp/script.js

```diff
@@ -55,8 +60,16 @@ function createListElement(text) {
  */
```

Comment:
Can you document what is shown when? e.g. in the function level comment mention that the login button is shown if the user is not logged in, and the comment form is shown otherwise!

Reply:
Done! Will keep working on my documentation.